src/AddressRegistry/StreetName/Municipality.cs

```diff
@@ -0,0 +1,22 @@
+namespace AddressRegistry.StreetName
+{
+    public class Municipality
```

Comment:
@ArneD Dient Municipality over te erven van een bepaalde class?

Reply:
Hmm, this could be moved to DataStructures maybe?

Reply:
Moved and renamed to MunicipalityData.